src/librustdoc/html/static/main.js

```diff
@@ -816,9 +823,10 @@
             output = '<h1>Results for ' + escape(query.query) +
                 (query.type ? ' (type: ' + escape(query.type) + ')' : '') + '</h1>' +
                 '<div id="titles">' +
-                '<div class="selected">Types/modules</div>' +
-                '<div>As parameters</div>' +
-                '<div>As return value</div></div><div id="results">';
+                `<div${currentTab === 0 ? ' class="selected"' : ""}>Types/modules</div>` +
+                `<div${currentTab === 1 ? ' class="selected"' : ""}>As parameters</div>` +
+                `<div${currentTab === 2 ? ' class="selected"' : ""}>As return value</div>` +
```

Comment:
I don't think we should use template literals?

* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility
* https://caniuse.com/#feat=template-literals

Reply:
Excellent point!